processor/metricstransformprocessor/README.md

```diff
@@ -0,0 +1,116 @@
+# Metrics Transform Processor <span style="color:red">**(UNDER DEVELOPMENT - NOT READY FOR USE)**</span>
+- <span style="color:red">ONLY support **single-batch processing** on metrics to avoid conflicts of different batches with the same time-series</span>
```

Comment:
What about saying something like this `This ONLY supports renames/aggregations **within individual metric batches**. It does not do any aggregation across batches, and so is not suitable for aggregating metrics from multiple sources (e.g. multiple nodes or clients), but only for aggregating metrics from a single source that groups its metrics for a particular time period into a single batch (e.g. host metrics from the VM the collector is running on).`

Reply:
This looks great!